MediaBrowser.Providers/Manager/MetadataService.cs

```diff
@@ -1142,6 +1142,15 @@ internal static void MergeBaseItemData(
             }
         }
 
+        private static bool IsYearOnlyDate(DateTime date)
+        {
+            return date > DateTime.MinValue && date < DateTime.MaxValue
+                && date.Year > 0
+                && date.Month == 1
+                && date.Day == 1
+                && date.TimeOfDay == TimeSpan.Zero;
+        }
```

Comment:
```suggestion
        => date.Month == 1 && date.Day == 1
```

No reason to check properties that we don't care about

Reply:
`0-0-0` isn't a valid DateTime in C# so no need to validate against it.
`DateTime.MinValue` is `0001-01-01`

Reply:
Updated, thanks for the review.